style.css

```diff
@@ -123,3 +123,188 @@ p {
   line-height: 32px;
   text-align: center;
 }
+
+.card {
+  height: 350px;
+  background-color: #ebf0ee;
+  flex-direction: column;
+  justify-items: center;
+  justify-content: space-around;
+  align-content: center;
+  border-radius: 6px;
+  display: grid;
+  margin-top: 20px;
+  margin-right: 25px;
+  margin-left: 25px;
+}
+
+h2 {
+  text-align: center;
+  color: #172b4d;
+}
```

Comment:
- [ ] I am highlighting this here because I can't make inline comments on the reset selector `* {}`. Kindly use the `font-family` of "Inter" for all text elements in this project. That is the font used in the template that you chose. 💻